wagtail/admin/views/pages/move.py

```diff
@@ -82,6 +96,20 @@ def move_confirm(request, page_to_move_id, destination_id):
         )
         action.execute()
 
+        if getattr(settings, 'WAGTAIL_I18N_ENABLED', False):
+            # Move translation and alias pages if they have the same parent page.
+            parent_page_translations = page_to_move.get_parent().get_translations()
```

Comment:
At first I was going to push for this logic to move to the hook, however on further reflection, it makes so much more sense to do it here as an additional defense mechanism. Good call 👍

Reply:
Thanks Dan! I admit I didn't do that on purpose, it just felt like an easier place to implement that :joy: